### DISCUSSION TITLE
Welcome to 8814au Discussions!

### DISCUSSION THREAD

**Discussion by morrownr** [morrownr/8814au#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others. Remember that this is a community we   build together 💪.

**Comment by Xpltr** [comment]
Morrownr, 
A while back, I had asked about putting the 8814au in the Kali Linux Repos.
I reached out  in 2021 to Kali devs, they placed your driver in the repos.
It was placed in the Kali repos in 2021. Just wanted you to know. I don't know if you were aware of this. 
Thanks for all your contributions.

Package: realtek-rtl8814au-dkms
Version: 5.8.5.1~git20220221.a538e38-0kali1
Priority: optional
Section: contrib/kernel
Maintainer: Kali Developers <devel@kali.org>
Installed-Size: 14.7 MB
Depends: dkms (>= 2.1.0.0)
Homepage: https://github.com/morrownr/8814au
Download-Size: 1,659 kB
APT-Manual-Installed: yes
APT-Sources: http://kali.download/kali kali-rolling/contrib amd64 Packages
Description: Realtek RTL8814AU driver in DKMS format
 This package provides the source code for RTL8814AU Linux driver (with monitor
 mode and frame injection) to be build with dkms. Kernel sources or headers are
 required to compile this module.
